sensor/common/scannerdefinitions/http_handler.go

```diff
@@ -20,29 +22,48 @@ var (
 	headersToProxy = set.NewFrozenStringSet("If-Modified-Since", "Accept-Encoding")
 )
 
-// scannerDefinitionsHandler handles requests to retrieve scanner definitions
+// ScannerDefinitionHandler handles requests to retrieve scanner definitions
 // from Central.
-type scannerDefinitionsHandler struct {
-	centralClient *http.Client
+type ScannerDefinitionHandler struct {
```

Comment:
why singular definition?

Reply:
It was a typo, thanks!

Reply:
Had to change it to just `Handler` due to style check:
```
sensor/common/scannerdefinitions/http_handler.go:29:6: exported: type name will be used as scannerdefinitions.ScannerDefinitionsHandler by other packages, and that stutters; consider calling this Handler
```